src/main/java/org/fxmisc/flowless/VirtualFlowContent.java

```diff
@@ -177,6 +186,44 @@ void scrollY(double deltaY) {
         orientation.scrollVertically(this, deltaY);
     }
 
+    void scrollXToPixel(double pixel) {
+        orientation.scrollHorizontallyToPixel(this, pixel);
+    }
+
+    void scrollYToPixel(double pixel) {
+        orientation.scrollVerticallyToPixel(this, pixel);
+    }
+
+    Val<Double> totalWidthEstimateProperty() {
+        return Val.wrap(orientation.widthEstimateProperty(this));
```

Comment:
Meaning if you look at ````orientation.widthEstimateProperty()````, you'll see that it returns ObservableValue<Double>:
````ObservableValue<Double> widthEstimateProperty(VirtualFlowContent<?, ?> content);````

So, when I write it as
````java
    Val<Double> totalWidthEstimateProperty() {
        return orientation.widthEstimateProperty(this);
    }
````

the IDE warns me with:
> Incompatibly Types:

>   Required: org.reactfx.value.Val<Double

>   Found: javafx.beans.value.ObservableValue<Double>